server/routes/posts.js

```diff
@@ -64,15 +66,19 @@ module.exports = function(app) {
 	});
 
 	//create a post
-	app.post('/api/posts', verifyUser, async (req, res) => {
+	//second parameter is uploading an array of files with multer with a limit of 10 pictures/videos for a post:
+	app.post('/api/posts', verifyUser, upload.array('media', 10), async (req, res) => {
 		if (!req.body) {
 			return res
 				.status(400)
 				.json({ message: `request is invalid` })
 				.end();
 		}
 
-		const post = { ...req.body, user: req.user.sub };
+		//iterating through all the files in the req.files array and extracting each file's path (which we need to send to the client) to the filePath variable:
+		const filePath = req.files.map(file => file.path)
+
+		const post = { ...req.body, media: filePath, user: req.user.sub };
```

Comment:
I really think we should start with single uploads until we are comfortable with AWS, and then try to implement multiple uploads.